appveyor.yml

```diff
@@ -32,7 +32,7 @@ before_build:
   - git submodule sync
   - git submodule update --init --recursive
   - SET INCLUDE=C:\Tools\vcpkg\installed\x64-windows\include;%INCLUDE%
-  - SET PATH=C:\Tools\vcpkg\installed\x64-windows\bin;%PATH%
+  - SET PATH=C:\Tools\vcpkg\installed\x64-windows\bin;C:\Strawberry\perl\bin;%PATH%
```

Comment:
Why is perl required?

Reply:
It is required to build OpenSSL that is a dependency of gRPC